air/src/air/transition/frame.rs

```diff
@@ -77,3 +79,64 @@ impl<E: FieldElement> EvaluationFrame<E> {
         &mut self.next
     }
 }
+
+// LAGRANGE KERNEL EVALUATION FRAME
+// ================================================================================================
+
+/// The evaluation frame for the Lagrange kernel.
+///
+/// The Lagrange kernel's evaluation frame is different from [`EvaluationFrame`]. Specifically,
+/// - it only contains evaluations from the Lagrange kernel column
+///   (compared to all rows in the case of [`EvaluationFrame`])
+/// - The column is evaluated at points `z`, `gz`, `g^2 z`, ..., `g^(2^(v-1)) z`,
+///   where `z` is an arbitrary point, and `g` is the trace domain generator
```

Comment:
We could just say that z is the OOD point to be more specific

Reply:
I see, it is used for both situations. Then I would probably use x instead of z in order to avoid confusions as z is, implicitly, used throughout the codebase to denote the OOD point.